src/docs/product/issues/states-triage/index.mdx

```diff
@@ -1,80 +1,60 @@
-- Resolved: The issue is marked resolved (or automatically resolved) because it was fixed or isn't expected to happen again. If the issue is seen again, its state changes to Unresolved.
+| Status     | Condition                                             | Custom Search Term |
+| ---------- | ----------------------------------------------------- | ------------------ |
+| New        | An issue that was created in the last 3 days.         | is:new             |
+| Ongoing    | An issue that was created more than 3 days ago.       | is:ongoing         |
+| Escalating | An issue that's exceeded its forecasted event volume. | is:escalating      |
+| Regressed  | A resolved issue that's come up again.                | is:regressed       |
+| Archived   | An issue that's been marked as archived.              | is:archived        |
+| Resolved   | An issue that's been marked as fixed.                 | is:resolved        |
 
-When any of the following conditions occur for an issue, that issue is flagged _for review_, included in the **Review List**, and given a label indicating why it was flagged:
+One way to sort the Issues that you see is by selecting a tab at the top of the page. On the “Unresolved tab” you'll find: New, Ongoing, Escalating, and Regressed issues, but you can also narrow down what you see by choosing the “For Review”, “New”, “Escalating”, “Ongoing”, or “Archived” tabs.
 
-- A new issue is created; it's labeled "New Issue"
-- A resolved issue re-occurs; it's labeled "Regression"
-- An ignored issue meets its ignore conditions; it's labeled "Unignored"
+![Manual vs. automatic issue status update](issue-status-docs.png)
 
-Issues in the **Review List**, or for-review issues, are a sub-set of all unresolved issues, and you can view these issues in the "For Review" tab of the **Issues** page. By default, they are filtered to display issues relevant to you; that is, issues that are assigned to you or suggested for you.
+## Manually Triaging Issues
 
-![For Review tab of the Issues page.](for-review-page-docs.png)
+While some Issue statuses are added and updated automatically, you can manually Archive or Resolve an Issue, which will also change its status.
 
-When you mark a for-review issue as reviewed, [sentry.io](https://sentry.io) removes the reason label and the issue remains in the Unresolved state.
+### Archive
 
-Resolving or ignoring an issue implicitly marks the issue reviewed.
+You can manually archive Issues that are less pressing or not applicable to you or your team. This will change their status to `Archived` and remove them from the Issues list. Sentry will automatically bring an Issue back to the top of the list and change its status to `Escalating` if the events in that Issue significantly increase over a short period of time. There's also an option to archive an issue for:
```

Comment:
```suggestion
You can manually archive Issues that are less pressing or not applicable to you or your team. This will change their status to `Archived` and remove them from the Issues list. Sentry will automatically bring an Issue back to the top of the list and change its status to `Escalating` if the events in that Issue significantly increase over a short period of time. There's also an option to mark an Issue as `Archived` for:
```